spark/src/main/java/org/apache/zeppelin/spark/SparkInterpreter.java

```diff
@@ -108,6 +108,12 @@
         .add("args", "", "spark commandline args")
         .add("zeppelin.spark.printREPLOutput", "true",
           "Print REPL output")
+        .add("spark.mahout", "false", "add Apache Mahout Support Support for Spark")
+        .add("spark.kryo.registrator",
+          "org.apache.mahout.sparkbindings.io.MahoutKryoRegistrator", "")
+        .add("spark.serializer", "org.apache.spark.serializer.KryoSerializer", "")
```

Comment:
While spark keeps `org.apache.spark.serializer.JavaSerializer` as a default value of `spark.serializer`, i think it's better Zeppelin keep the same default value.

How about apply `spark.serializer` and `spark.kryo.registrator` property programmatically when `spark.mahout` is true?